packages/shared/src/components/fields/Dropdown.tsx

```diff
@@ -124,9 +127,16 @@ export function Dropdown({
   const fullScreen = openFullScreen ?? isMobile;
 
   return (
-    <div
-      className={classNames(styles.dropdown, className.container)}
-      {...props}
+    <ConditionalWrapper
```

Comment:
Believe it or not, without this, there is a crazy bug going on, the classes used here are magically passed to some other elements 😭

Reply:
Also love to know the issue, feels like it's not needed 😅

Reply:
I will do a loom recording to explain it easier 😭

Reply:
🤣